__tests__/typescript/class-names-to-type-definitions.test.ts

```diff
@@ -1,13 +1,17 @@
 import { classNamesToTypeDefinitions, ExportType } from "../../lib/typescript";
 
-describe("classNamesToTypeDefinitions", () => {
+jest.mock("../../lib/prettier/can-resolve", () => ({
+  canResolvePrettier: () => false,
+}));
+
+describe("classNamesToTypeDefinitions (without Prettier)", () => {
```

Comment:
Probably should test with Prettier as well, but that logic is tested in other areas so I'm not overly concerned.

We actually should eliminate most of this custom handling in here (eg: `quoteType`) and expect people to use Prettier for this.